README.md

```diff
@@ -386,6 +385,54 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered with lifecycle methods first, then by class inheritance. A `pragma mark -` is used to separate each group.
+
+“Lifecycle methods” are roughly defined, for the purpose of ordering, as those methods dealing with the creation or destruction of instances: initializers, `+alloc`, `-dealloc`, `-initialize`, `-load`, factory methods, `NSCoding` methods, and the like.
```

Comment:
@mattbischoff I'll fix `-initialize` and `-load` in a bit, but don't want to (visually) collapse your other comments on this discussion yet so I will hold off on pushing that change